apps/web/src/components/layouts/AppLayout/Header/Breadcrumb/Evaluations/index.tsx

```diff
@@ -25,14 +26,17 @@ export function EvaluationBreadcrumbItems({
 
   const options = useMemo<BreadcrumbSelectorOption[]>(() => {
     if (!evaluations) return []
-    return evaluations.map((p) => ({
-      label: p.name,
-      href: segments[1]
-        ? ROUTES.evaluations.detail({ uuid: p.uuid })[
-            segments[1] as EvaluationRoutes
-          ].root
-        : ROUTES.evaluations.detail({ uuid: p.uuid }).root,
-    }))
+    return evaluations.map((p) => {
+      const baseRoute = ROUTES.evaluations.detail({ uuid: p.uuid })
+      const href = evaluationSegment
+        ? (baseRoute[evaluationSegment]?.root ?? baseRoute.root)
+        : baseRoute.root
+
+      return {
+        label: p.name,
+        href,
+      }
+    })
```

Comment:
Fixed a bugfix where the Breadcrumb component failed on `/evaluations/<uuid>/destroy`